login_server/player/loadouts.py

```diff
@@ -129,9 +168,9 @@ class Loadouts:
     loadout_key2id = {v: k for k, v in loadout_id2key.items()}
 
     def __init__(self):
-        self.loadout_dict = self.defaults()
+        self.loadout_dict = self.defaults(default_loadouts)
 
-    def defaults(self):
+    def defaults(self, default_loadout_defs):
```

Comment:
Put the responsibility for determining the default loadouts in the defaults function. So remove the default_loadouts variable and the "if" it contains and don't pass this variable to the defaults function.

Reply:
Done.